src/NuGet.Clients/NuGet.CommandLine/NuGetResources.resx

```diff
@@ -6182,4 +6182,19 @@ Oluşturma sırasında NuGet'in paketleri indirmesini önlemek için, Visual Stu
     <comment>Do not localize project.json and `PackageReference` 
 0 - project path (project.json file path)</comment>
   </data>
+  <data name="CommandDeprecationWarningAlternative" xml:space="preserve">
+    <value>'{0} {1}' is deprecated. Use '{0} {2}' instead</value>
+    <comment>{0} - name of the assembly running the command, e.g., NuGet
```

Comment:
This only runs in NuGet.exe. 
The parameter for 0 the assembly is an overkill.

Reply:
I disagree because I rename all my `nuget.exe`s to have the version number in the filename, so I can have them all side-by-side in my path. 😄

Reply:
I think that's a marginal benefit. Like you literally just ran the command. 

Its fine as is, which is why I marked this as resolved